### ISSUE TITLE
Error saving Database

### ISSUE THREAD

**Issue by dx05** [keepassxreboot/keepassxc#5721]
I use Keepassxc from two mac accounts with same database (alwa
ys worked). Now I tried to change an entry from secondary account and keepassxc says "Schreiben der Datenbank fehlgeschlagen. Die Zieldatei existiert bereits. Sicherungskopie der Datenbank ist ....". I did not activate the database backup option in the settings. Write access is granted for both accounts.

<img width="1042" alt="Bildschirmfoto 2020-11-19 um 12 41 43" src="https://user-images.githubusercontent.com/40899951/99662996-448c2c80-2a66-11eb-9eb2-a5b3a4cf3e86.png">

Kind regards, dx05

**Comment by phoerious** [comment]
Are you trying to save to a remote folder or anything or is there a sync service (Dropbox, Nextcloud) etc. running in the background?

**Comment by dx05** [comment]
Hi,
no nothing remote, it’s a file in the shared user folder on the local machine, no dropbox or so running…
Kind regards, dx05

**Comment by droidmonkey** [comment]
That error will appear if you do not have write access to the folder or the destination kdbx file is locked for writing. Reboot the Mac.

**Comment by dx05** [comment]
But both user accounts have read+write permissions
The error file seemed to be a backup file, not the main database file, as shown in the attached picture.

**Comment by dx05** [comment]
PS: The database file is always only opened in one account at the same time.

**Comment by droidmonkey** [comment]
I'm just telling you the circumstances this error will appear. Clearly you are hitting these cases, did you reboot? Have you updated macOS recently, that could cause permissions problems. Can you reproduce this with a new database?

**Comment by dx05** [comment]
Hi,
I’ve done as you told me, restartet Mac. Didn’t change anything. Then I created a new new database in aaccount Ä and added one entry, located in shared user folder. Then I quit Keepassxc. Then I added read/write permission to the database file for account B (per finder info dialog). Then I logged in account B and startet Keepassxc, opened the test database and modified the entry. Same probblem occurred, can’t save the change.

Kind Regards, dx05 (Harald)
